OracleTuxedo/core/samples/ws_ssl_svr/ws_svr_runme.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+# 
+# Shell script to build and run ws_svr.  It assume TUXDIR has been set, or that 
+# Tuxedo has been installed to: ~/tuxHome/tuxedo12.1.3.0.0   If not, invoke 
+# this script with a single argument indicating the location of TUXDIR.
+#
+# Usage: source ws_svr_runme.sh
+#
+mkdir -p /u01/oracle/user_projects
+cd /u01/oracle/user_projects || return 1
+rm -rf ws_svr
+mkdir ws_svr
+cd ws_svr || return 1
+
+# Create environment setup script setenv.sh
+HOSTNAME=$(uname -n)
+APPDIR=$(pwd)
+export HOSTNAME APPDIR
+export WS_PORT=9055
+export SSL_PORT=9060
+#export TUX_LOADBAL_IP="138.3.79.82"
+export SEC_PRINCIPAL_NAME="ISH_tuxqa"            # ssl
+
+cat >setenv.sh << EndOfFile
+source ${TUXDIR}/tux.env
+export HOSTNAME=${HOSTNAME}
+export APPDIR=${APPDIR}
+export TUXCONFIG=${APPDIR}/tuxconfig
+export TM_SECURITY_CONFIG=NONE
+export TM_ALLOW_NOTLS=Y
+export SSL_PORT=${SSL_PORT}                      # ssl
+export ULOG_SSLINFO=yes                          # ssl
+export TM_MIN_PUB_KEY_LENGTH=1024                # ssl
+export SEC_PRINCIPAL_NAME=${SEC_PRINCIPAL_NAME}  # ssl
+export PASSVAR=tuxsvr123                         # ssl
```

Comment:
```suggestion
```

Oracle does not permit the use of hard-coded passwords. You will need to randomly generate a password using something like `$(openssl rand -hex 8)` and then output that to the user. This password should then be passed back into the container using either Docker or Podman secrets or via an environment variable.